src/baseCanvas/Canvas.tsx

```diff
@@ -35,7 +35,7 @@ export interface Props {
 }
 export class BaseCanvas extends Component<Props> {
   private name: string;
-  private canvas: HTMLCanvasElement;
+  public canvas: HTMLCanvasElement;
```

Comment:
What do you need this for? You can actually get a read-only version of the canvas from the canvas context (which is already public) if you just need things like height and width or something?

Reply:
ah i don't think i do need it now, I'll change it back